huxley/core/admin/delegate.py

```diff
@@ -5,9 +5,10 @@
 
 from django.conf.urls import patterns, url
 from django.contrib import admin
-from django.http import HttpResponse
+from django.core.urlresolvers import reverse
+from django.http import HttpResponse, HttpResponseRedirect
 
-from huxley.core.models import Delegate
+from huxley.core.models import Assignment, Committee, Country, Delegate, School
```

Comment:
Doesn't look like the Committee, Country, and School classes are used in this file.